sdk/resourcemanager/compute/armcompute/autorest.md

```diff
@@ -4,9 +4,10 @@
 
 ``` yaml
 azure-arm: true
+tag: package-2021-12-01
 require:
-- https://github.com/Azure/azure-rest-api-specs/blob/cc26f1d28c939bbcde00a29aa8958b3eebb10cea/specification/compute/resource-manager/readme.md
-- https://github.com/Azure/azure-rest-api-specs/blob/cc26f1d28c939bbcde00a29aa8958b3eebb10cea/specification/compute/resource-manager/readme.go.md
+- https://github.com/Alancere/azure-rest-api-specs/blob/mocktest_compute/specification/compute/resource-manager/readme.md
```

Comment:
could you explain the reason for this change?

Reply:
This pull request is mainly used for testing